coding-concepts.js

```diff
@@ -7,67 +7,60 @@
 
 // --------------------1) What will this log?
 
-var cohort = "Alpha 2022"
-// console.log(cohort.split(""))
-
-// a) Your answer:
-// b) Verify and explain:
+var cohort = "Alpha 2022";
+console.log(cohort.split(""));
 
+// a) Your answer: ['A', 'L', 'P', 'H', 'A', ' ', '2', '0', '2', '2']
+// b) Verify and explain: The use of .split when console logging will return an array separated individually by character
 
 // --------------------2) What will this log?
 
 const greeter = (name) => {
-  `Hello, ${name}!`
-}
-// console.log(greeter("LEARN Student"))
-
-// a) Your answer:
-// b) Verify and explain:
+  `Hello, ${name}!`;
+};
+console.log(greeter("LEARN Student"));
 
+// a) Your answer: Undefined
+// b) Verify and explain: Undefined because there is no "return" in front of line 19 giving the code instructions to return "Hello, LEARN Student!"
 
 // --------------------3) What will this log?
 
-var multBy2 = [4, 5, 6, 7, 8].map(value => value * 2)
-// console.log(multBy2)
-
-// a) Your answer:
-// b) Verify and explain:
+var multBy2 = [4, 5, 6, 7, 8].map((value) => value * 2);
+console.log(multBy2);
 
+// a) Your answer: [8, 10, 12, 14, 16]
+// b) Verify and explain: The use of ".map" pulls in the the var multby2, while as the function and use of "value * 2" takes the numbers from our original array and multiples them by 2.
 
 // --------------------4) What will this log?
 
-var onlyOdds = [11, 12, 13, 14, 15].filter(value => value % 2 !== 0)
-// console.log(onlyOdds)
-
-// a) Your answer:
-// b) Verify and explain:
+var onlyOdds = [11, 12, 13, 14, 15].filter((value) => value % 2 !== 0);
+console.log(onlyOdds);
 
+// a) Your answer: [11, 13, 15]
+// b) Verify and explain: the use of ".filter" allows us to call a function in which we are able to return specific values where in this class are only the odd numbers of our original array.
```

Comment:
Filter is a higher-order function that has a built-in if-else statement.  Its job is to evaluate a condition and give back an array of only that which has met the condition.